sdk/nodejs/sdk/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@openziti/zrok",
+  "version": "0.1.0",
```

Comment:
Why is this still `0.1.0`?

Reply:
The `version` gets auto-updated by the GH Action that runs whenever the `main` branch gets updated, and the semver will become whatever the prevailing git `tag` is at the time.  The package that gets published to NPM will have the same version as the `zrok` repo's git `tag`.